direnv/config.nu

```diff
@@ -0,0 +1,16 @@
+## To enable direnv in nushell we must run code in the context of the current shell - i.e it cannot be within a block and it needs to run as a "code" string as per https://github.com/nushell/nushell/pull/5982 (so you must run nushell 0.66 or later). That way it works as if you'd typed in and run the code directly in your shell.
+## Direnv knows what to do otherwise and will export the env to load (or unload) based on what is in your current environment so this is all that is needed with some checks for empty strings, defaulting then to an empty table so that load-env is always happy.
+
+let-env config = {
+  hooks: {
+    env_change: {
+      PWD: [{
+        code: "
+          let direnv = (direnv export json | from json)
```

Comment:
Would null-terminated env vars like in https://github.com/direnv/direnv/pull/973 be potentially more efficient?

Reply:
That looks harder to parse to me.

Reply:
If the json parser is written is in rust this hardly makes any difference given that direnv executes slow shell scripts...